x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -333,21 +281,16 @@ export function getCloneFormStateFromJobConfig(
       ? analyticsJobConfig.source.index.join(',')
       : analyticsJobConfig.source.index,
     modelMemoryLimit: analyticsJobConfig.model_memory_limit,
-    excludes: analyticsJobConfig.analyzed_fields.excludes,
+    includes: analyticsJobConfig.analyzed_fields.includes,
   };
 
-  if (
-    isRegressionAnalysis(analyticsJobConfig.analysis) ||
-    isClassificationAnalysis(analyticsJobConfig.analysis)
-  ) {
-    const analysisConfig = analyticsJobConfig.analysis[jobType];
+  const analysisConfig = analyticsJobConfig.analysis[jobType];
 
-    for (const key in analysisConfig) {
-      if (analysisConfig.hasOwnProperty(key)) {
-        const camelCased = toCamelCase(key);
-        // @ts-ignore
-        resultState[camelCased] = analysisConfig[key];
-      }
+  for (const key in analysisConfig) {
+    if (analysisConfig.hasOwnProperty(key)) {
+      const camelCased = toCamelCase(key);
+      // @ts-ignore
```

Comment:
I see this ignore was here in the previous code too, still, can you add a comment why it's happening?